NuRadioReco/modules/io/RNO_G/readRNOGDataMattak.py

```diff
@@ -724,30 +735,27 @@ def run(self):
 
         evt: `NuRadioReco.framework.event.Event`
         """
-        event_idx = -1
+        self._event_idx = 0
         for dataset in self._datasets:
             dataset.setEntries((0, dataset.N()))
 
+            # _filter_event is a wrapper for the selectors defined in begin()
+            selectors = [self._filter_event, self._check_for_valid_information_in_event_info] if len(self._selectors) !=0 \
+                   else [self._check_for_valid_information_in_event_info]
+
```

Comment:
should those be merged with the selectors you can pass as argument to this class?

Reply:
What do you mean?

Reply:
Should they be merged with 
```       
selectors: list of Callables
      List of Callable(eventInfo) -> bool to pass to mattak.Dataset.iterate to select events.
```

Reply:
Ow yeah that can cause confusion I have renamed it